Configuration/Geometry/README.md

```diff
@@ -85,25 +85,11 @@ The script also handles the common and forward elements of the geometry:
 * F6: same as F4 with modifications needed for BRM and forward shield
 
 Several detector combinations have been generated:
-* D49 = T15+C9+M4+I10+O4+F2
-* D50 = T15+C9+M4+I11+O4+F2
-* D60 = T15+C10+M4+I10+O4+F3
-* D64 = T22+C11+M4+I11+O5+F4
-* D65 = T23+C11+M4+I11+O5+F4
-* D66 = T21+C11+M8+I11+O5+F4
-* D67 = T21+C11+M9+I11+O5+F4
-* D68 = T21+C11+M6+I11+O5+F4
-* D69 = T21+C12+M6+I11+O5+F5
-* D70 = T21+C13+M7+I11+O6+F6
-* D71 = T21+C14+M7+I11+O7+F6
-* D72 = T21+C11+M6+I12+O5+F4
-* D74 = T21+C14+M9+I11+O7+F6
-* D75 = T21+C14+M7+I13+O7+F6
+* D49 = T15+C9+M4+I10+O4+F2 (HLT TDR baseline)
+* D60 = T15+C10+M4+I10+O4+F3 (With HFNose)
```

Comment:
@srimanob
we should profit to update the D60 here:
i.e. we can pair the F3 of the HFnose with the more up to date geometries for the T,C,M,I,O configurations.

Reply:
Hi @mariadalfonso 
Thanks for the comment.
Can it be paired easily, or need some extra work? I would prefer we do it in a separate PR after this one.

Reply:
I think they can be paired easily, but @bsunanda knows better

Reply:
The new detector configuration with HFNose will be added after this PR is merged.